lib/irc.js

```diff
@@ -93,6 +94,23 @@ function Client(server, nick, opt) {
       self.connect();
     }
 
+    if (self.opt.autoEncoding) {
+        var charsetDetector = require('node-icu-charset-detector');
+        var Iconv = require('iconv').Iconv;
+
+        self.convertEncoding = function(str) {
```

Comment:
What do you think about stubbing out `Client.prototype.convertEncoding` to be a no-op in case it is every called and `Client.opt.autoEncoding` is false?